### PR TITLE
New keyboard: 0-Sixty

### PR DESCRIPTION
**Pull request by vinamarora8**
Adds support for a new keyboard: 0-Sixty, a 60 key ortholinear keyboard based on the ProMicro.
0-Sixty

## Description

A new directory called `0_sixty` is created inside `keyboards/`. To this, all necessary files are added, along with 3 keymaps: default, via, and a personal one.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/0_sixty/0_sixty.h [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+) \
+{ \
+    { k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b }, \
+    { k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b }, \
+    { k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b }, \
+    { k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b }, \
+    { k40, k41, k42, k43, k44, k45, k46, k47, k48, k49, k4a, k4b }  \
+}
+
+#define LAYOUT_1x2uL(						\
```
Comment (fauxpark) [inline keyboards/0_sixty/0_sixty.h:39]:
```suggestion
#define LAYOUT_1x2uL( \
```

File: keyboards/0_sixty/readme.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# 0-Sixty
+
+![0-Sixty](https://imgur.com/azmMb6R.jpg)
```
Comment (fauxpark) [inline keyboards/0_sixty/readme.md:3]:
```suggestion
![0-Sixty](https://imgur.com/azmMb6Rl.jpg)
```

File: keyboards/0_sixty/readme.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+![0-Sixty PCB](https://i.imgur.com/I0TkQ0R.jpg)
```
Comment (fauxpark) [inline keyboards/0_sixty/readme.md:5]:
```suggestion
![0-Sixty PCB](https://i.imgur.com/I0TkQ0Rl.jpg)
```

File: keyboards/0_sixty/readme.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+0-Sixty (pronounced "zero to sixty") is a 60 key ortholinear keyboard
+inspired by the
+[Discipline](https://github.com/coseyfannitutti/discipline) and the
+[Preonic](https://drop.com/buy/preonic-mechanical-keyboard). It is
+designed as a DIY through-hole kit and uses the ProMicro as its
+controller.
```
Comment (fauxpark) [inline keyboards/0_sixty/readme.md:12]:
```suggestion
0-Sixty (pronounced "zero to sixty") is a 60 key ortholinear keyboard inspired by the [Discipline](https://github.com/coseyfannitutti/discipline) and the [Preonic](https://drop.com/buy/preonic-mechanical-keyboard). It is designed as a DIY through-hole kit and uses the ProMicro as its controller.
```

File: keyboards/0_sixty/readme.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+* Keyboard Maintainer: [vinamarora8](https://github.com/vinamarora8)
+* Hardware Supported: 0-Sixty PCB
+* Hardware Availability: Private Group-Buy
+
+Getting the board into bootloader mode:
+
+To be able to flash firmware onto this board, you'll need to bring the
+ProMicro into bootloader mode. Simply tap the small reset button
+twice. The button is placed to the right of the ProMicro.
```
Comment (fauxpark) [inline keyboards/0_sixty/readme.md:24]:
```suggestion
To be able to flash firmware onto this board, you'll need to bring the ProMicro into bootloader mode. Simply tap the small reset button twice. The button is placed to the right of the ProMicro.
```

File: keyboards/0_sixty/rules.mk [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes           # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (fauxpark) [inline keyboards/0_sixty/rules.mk:15]:
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by vinamarora8** [comment]
Thanks a lot! :)

**Pull request merged**